gitlab-pipeline/stage/test.yml

```diff
@@ -233,10 +245,6 @@ test:backend-integration:azblob:enterprise:
         sleep 1
       done
 
-    # Set minimaly required by Opensearch 'max virtual memory areas'
```

Comment:
Q: I don't see why this should be moved as a part of this commit

Reply:
Just keeping them together, I suppose.

Reply:
Yes, just keeping them together on the every beginning of the job execution.